src/edu/csus/ecs/pc2/core/imports/clics/CLICSJsonUtilities.java

```diff
@@ -278,8 +278,8 @@ public static List<String> getTeamIdsByRank(List<TeamScoreRow> scoreRows, int lo
                 teamIds.add(teamScoreRow.getTeam_id() + "");
             }
         }
-
-        return teamIds;
+        
+        return (String[]) teamIds.toArray(new String[teamIds.size()]);
```

Comment:
Related to my comments on class `CLICSAward`, this line of code exposes the internal storage type (`String[]`) to external clients.  External clients should know only that they are getting a `List`.

Note that reverting this line also requires reverting all the above changed lines (back to what they were).